spec/integration/root_login_spec.rb

```diff
@@ -15,7 +15,7 @@
     follow_redirect!
     expect(request.fullpath).to eq(root_path)
     # Adjust according to new start-page
-    expect(response.body).to match(/Hi  Root! Want to recover a password?/)
+    expect(response.body).to match(/<div id='ember'><\/div>/)
```

Comment:
Hmm inside the start-page PR, I've adressed it to just check if it's being redirected to the frontend... Don't really know what the better option is... Do you have a preference?

Reply:
Not really. What do you prefer? I think that the div is easier to match for capybara than a regex text.